res/css/views/context_menus/_IconizedContextMenu.scss

```diff
@@ -90,6 +90,7 @@ limitations under the License.
                 width: 16px;
                 min-width: 16px;
                 max-width: 16px;
+                mask-size: contain;
```

Comment:
The icon itself doesn't bear a mask, that's applied to the `::before` element - the bug here is actually that the mask is being applied to the wrong element - See https://github.com/matrix-org/matrix-react-sdk/blob/8fe582b094c73a6645fc2023da4cdce0c567185a/res/css/views/rooms/_RoomList.scss#L36-L41 and compare it to the mask-images above